spec/support/api_spec_helper.rb

```diff
@@ -89,14 +89,13 @@ def define_user
     @user  = FactoryGirl.create(:user,
                                 :name             => api_config(:user_name),
                                 :userid           => api_config(:user),
-                                :password_digest  => BCrypt::Password.create(api_config(:password)),
+                                :password         => api_config(:password),
                                 :miq_groups       => [@group],
                                 :current_group_id => @group.id)
   end
 
   def init_api_spec_env
-    MiqRegion.seed
-    MiqDatabase.seed
+    @miq_database = FactoryGirl.create(:miq_database)
```

Comment:
The MiqDatabase.seed should be almost nearly as performant as the FactoryGirl.create, so let's put that back (plus there was a useless assigment, so no need for that).  The MiqRegion.seed removal is a good find though.